docs/source/cli_help/list.rst

```diff
@@ -0,0 +1,22 @@
+.. role:: bash(code)
+
+.. _list:
+
+list
+----
+Usage:
+
+.. code-block:: shell
+
+    git machete list <category>
+where <category> is one of: :bash:`addable`, :bash:`managed`, :bash:`slidable`, :bash:`slidable-after <branch>`, :bash:`unmanaged`, :bash:`with-overridden-fork-point`.
+
+Lists all branches that fall into one of the specified categories:
+    * :bash:`addable`: all branches (local or remote) than can be added to the definition file,
+    * :bash:`managed`: all branches that appear in the definition file,
+    * :bash:`slidable`: all managed branches that have an upstream and can be slid out with :bash:`slide-out` command
```

Comment:
Link

Reply:
I'm assuming that you mean for both file and slid-out.

Reply:
Mostly for `slide-out` TBH

Reply:
Ok, I will unlink file.